pluwum/react-shoppinglist
# #153717360 User should see indication when data loading is delayed

#### What does this PR do?
The PR Add a spinner and a message as shopping list or item list loads.

#### How should this be manually tested?
 - After setting up the application, add some entries for shopping lists and under them add items.  
 - Navigate away to any other page. 
-  Enable bandwidth throttling on your browser to mock slow network connection and navigate back to the shopping list view page.
- As the page load, a spinner should be visible.

#### What are the relevant pivotal tracker stories?
[#153717360](https://www.pivotaltracker.com/n/projects/2127250/stories/153717360)
